x-pack/plugins/watcher/public/application/app_context.tsx

```diff
@@ -16,18 +16,14 @@ interface ContextValue extends Omit<AppDeps, 'docLinks'> {
 
 const AppContext = createContext<ContextValue>(null as any);
 
-// eslint-disable-next-line @typescript-eslint/naming-convention
-const generateDocLinks = ({ ELASTIC_WEBSITE_URL, DOC_LINK_VERSION }: DocLinksStart) => {
-  const elasticDocLinkBase = `${ELASTIC_WEBSITE_URL}guide/en/`;
-  const esBase = `${elasticDocLinkBase}elasticsearch/reference/${DOC_LINK_VERSION}`;
-  const kibanaBase = `${elasticDocLinkBase}kibana/${DOC_LINK_VERSION}`;
-  const putWatchApiUrl = `${esBase}/watcher-api-put-watch.html`;
-  const executeWatchApiUrl = `${esBase}/watcher-api-execute-watch.html#watcher-api-execute-watch-action-mode`;
-  const watcherGettingStartedUrl = `${kibanaBase}/watcher-ui.html`;
+const generateDocLinks = ({ links }: DocLinksStart) => {
+  const putWatchApiUrl = `${links.apis.putWatch}`;
+  const executeWatchApiUrl = `${links.apis.executeWatchActionModes}`;
+  const watcherGettingStartedUrl = `${links.watcher.ui}`;
   const watchActionsConfigurationMap = {
-    [ACTION_TYPES.SLACK]: `${esBase}/actions-slack.html#configuring-slack`,
-    [ACTION_TYPES.PAGERDUTY]: `${esBase}/actions-pagerduty.html#configuring-pagerduty`,
-    [ACTION_TYPES.JIRA]: `${esBase}/actions-jira.html#configuring-jira`,
+    [ACTION_TYPES.SLACK]: `${links.watcher.slackAction}`,
+    [ACTION_TYPES.PAGERDUTY]: `${links.watcher.pagerDutyAction}`,
+    [ACTION_TYPES.JIRA]: `${links.watcher.jiraAction}`,
```

Comment:
i noticed when testing this that the `#configuring-x` hash was removed from the end of the URL after these changes were applied. Do you think we should re-add them @jrodewig ?

Reply:
Thanks @jloleysens. This is intentional. I think the links are better without the anchor. The intro of each action page contains requirements that apply to both UI and API users. For example, in the [Slack action docs](https://www.elastic.co/guide/en/elasticsearch/reference/current/actions-slack.html):

<img width="715" alt="Screen Shot 2021-03-15 at 11 14 08 AM" src="https://user-images.githubusercontent.com/40268737/111176542-cfe27e00-857f-11eb-8226-ea62242e4340.PNG">

Linking to the `#configuring-x` "skips" this intro. The `#configuring-x` sections just provide an API equivalent of the UI configuration. I don't think it'll be helpful in most cases.

<img width="772" alt="Screen Shot 2021-03-15 at 11 13 59 AM" src="https://user-images.githubusercontent.com/40268737/111176764-00c2b300-8580-11eb-83af-d419f3f4f4b4.PNG">